### PR TITLE
[server] Lift ratelimit on sendHeartBeat

### PR DESCRIPTION
**Pull request by geropl**
## Description

Not necessary anymore since we rolled out https://github.com/gitpod-io/ops/pull/8744.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/16880

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/server/src/auth/rate-limiter.ts [hunk 1]
```diff
@@ -260,10 +261,6 @@ function getConfig(config: RateLimiterConfig): RateLimiterConfig {
             points: 10,
             durationsSec: 10,
         },
-        sendHeartBeat: {
```
Comment (easyCZ) [inline components/server/src/auth/rate-limiter.ts:263]: Let's keep it there, but with a much higher value as a safe-guard. Something like 100 per user, which would require you to have ~50 workspaces

Comment (geropl) [reply components/server/src/auth/rate-limiter.ts:263]: Done.

(Maybe that value should be [the default](https://github.com/gitpod-io/gitpod/blob/03ceb6dfcc8b24350133ec4a9eb2ebf0e873faa1/components/server/src/auth/rate-limiter.ts#L248-L251) in the first place :see_no_evil: )

Comment (easyCZ) [reply components/server/src/auth/rate-limiter.ts:263]: Yeah, probably. Let's fix in a follow-up.

**Review by easyCZ — APPROVED** [review]

File: components/server/src/auth/rate-limiter.ts (hunk 1 above)
Comment (geropl) [reply components/server/src/auth/rate-limiter.ts:263]: > Let's fix in a follow-up.

Issue here: https://github.com/gitpod-io/ops/issues/8812

**Pull request merged**